CHANGELOG.en-US.md

```diff
@@ -15,6 +15,33 @@ timeline: true
 
 ---
 
+# 3.2.0
+
+`2018-02-04`
+
+- 🌟 Add new `tabBarGutter` prop to Tab to allow setting gutter between tabs. [#8644](https://github.com/ant-design/ant-design/pull/8644) [@lixiaoyang1992](https://github.com/lixiaoyang1992)
+- 🌟 Add new `hasSider` prop to Layout to avoid render error when server rendering. [#8937](https://github.com/ant-design/ant-design/issues/8937)
+- 🌟 Add new `successPercent` to Progess to alow showing two phases of progess. [Demo](https://ant.design/components/progress/#components-progress-demo-segment)
```

Comment:
to Progess to alow showing two phases of progess -> to Progress to allow showing two phases